index.js

```diff
@@ -1,46 +1,71 @@
+require("./messages");
+var steam = require("steam");
+
+const DOTA_APP_ID = 570;
+
 var EventEmitter = require('events').EventEmitter,
     fs = require("fs"),
+    path = require("path"),
     util = require("util"),
     bignumber = require("bignumber.js"),
-    Schema = require('protobuf').Schema,
-    base_gcmessages = new Schema(fs.readFileSync(__dirname + "/generated/base_gcmessages.desc")),
-    gcsdk_gcmessages = new Schema(fs.readFileSync(__dirname + "/generated/gcsdk_gcmessages.desc")),
-    dota_gcmessages = new Schema(fs.readFileSync(__dirname + "/generated/dota_gcmessages.desc")),
-    protoMask = 0x80000000,
+    ProtoBuf = require('protobufjs'),
     Dota2 = exports;
 
+ProtoBuf.convertFieldsToCamelCase = true;
```

Comment:
Alright, I'll look into refactoring the code-base this weekend, the README might take some more time :p